### PR TITLE
[web][image-picker] Fix `base64` return

### PR DESCRIPTION
**Pull request by simonezuccala**
# Why
`ImagePicker` doesn't return the `base64` of the image as stated in the [documentation](https://docs.expo.io/versions/latest/sdk/imagepicker/#returns-4).

# How
The base64 of the image is returned if the `base64` param is passed as `true` to `readFile()` (which is called by `openFileBrowserAsync()`).

# Test Plan
- Navigate to `cd apps/bare-expo`
- Run `yarn web`
- Press on the _APIs_ tab
- Press on _ImagePicker_
- Toggle the _With base64_ switch
- Press on _Pick photo_ or any other option that opens the file browser for image selection
- Select an image, then confirm
- You should see the base64 for the selected image in the object under the image and in the console inside _Results_ 

### Before
Screenshot 2021-04-14 at 09 32 18

### After
Screenshot 2021-04-14 at 09 30 06

### CODE REVIEW

File: packages/expo-image-picker/src/ExponentImagePicker.web.ts [hunk 1]
```diff
@@ -150,6 +157,7 @@ function readFile(targetFile: Blob): Promise<ImageInfo> {
             uri,
             width: image.naturalWidth ?? image.width,
             height: image.naturalHeight ?? image.height,
+            ...(options.base64 && { base64: uri.substr(uri.indexOf(',') + 1) }),
```
Comment (misterdev) [inline packages/expo-image-picker/src/ExponentImagePicker.web.ts:160]: From the `FileReader` [doc](https://developer.mozilla.org/en-US/docs/Web/API/FileReader/readAsDataURL)

Screenshot 2021-04-14 at 12 01 19

Comment (lukmccall) [reply packages/expo-image-picker/src/ExponentImagePicker.web.ts:160]: Could you add this note as a comment in the code?

**Review by lukmccall — CHANGES_REQUESTED** [review]
Thanks for your contribution 🥇

Looks good to me 💪

Please, add the changelog entry to `expo-image-picker` package.

File: packages/expo-image-picker/src/ExponentImagePicker.web.ts (hunk 1 above)
Comment (simonezuccala) [reply packages/expo-image-picker/src/ExponentImagePicker.web.ts:165]: Sure, I added it.

**Review by bbarthec — APPROVED** [review]
@simonezuccala, thank you for your contribution 🎉 😊 
The code looks fine to me after all reviews-based changes you've applied 💪 
I'm merging it now 😉

**Review by bbarthec — CHANGES_REQUESTED** [review]
Whoopsie 🤕 
Before merging I need you to rebuild the `expo-image-picker`, because build files went out of sync with the sources ... 😞

**Comment by simonezuccala** [comment]
> Whoopsie 🤕
> Before merging I need you to rebuild the `expo-image-picker`, because build files went out of sync with the sources ... 😞

Oops, my bad! I've pushed the rebuilt files.

**Review by bbarthec — APPROVED** [review]
LGTM 💪

**Pull request merged**